JAX to HLO/jax_to_optimized_hlo_example.py

```diff
@@ -0,0 +1,31 @@
+from jax import numpy, grad
+from jax.lib import xla_client
+from jax_to_hlo import jax_to_hlo
+
+'''
+This file is an example of how to get optimized HLO from a JAX program.
```

Comment:
Bonus points for converting this to a unit test, so that some later maintainer can easily find out what's working and what's not.